resources/lang/en/people.php

```diff
@@ -258,6 +258,7 @@
     'activities_add_success' => 'The activity has been added successfully',
     'activities_update_success' => 'The activity has been updated successfully',
     'activities_delete_success' => 'The activity has been deleted successfully',
+    'activities_who_was_involved' => 'Who was involved?',
```

Comment:
Could you also add this key to the other languages files (you can copy/paste the EN translation, but at least the key will be there for other translators to translate it)?